doc/api/prev_api_changes/api_changes_0.54.rst

```diff
@@ -92,7 +92,7 @@ Axes creation
     use::
 
       f = Figure(figsize=(5,4), dpi=100)
-      a = f.add_subplot(111)
+      a = f.add_subplot(1, 1, 1)
```

Comment:
I strongly think we should not change old API changes notes -- they should stay representative of what the lib was back when that release occured.

Reply:
Agreed.  Well, I kept a couple of the really recent ones...